README.md

```diff
@@ -0,0 +1,63 @@
+# GA4GH SDK
+
+## Synopsis
+
+A Generic SDK and CLI for GA4GH API services
+
+## Basic usage
+
+## Installation
+
```

Comment:
Please add "Coming soon..." or "TBA" instead of just leaving the sections empty.